pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/physicaldigital/SierraPhysicalDigitalMergeRule.scala

```diff
@@ -8,37 +8,31 @@ import uk.ac.wellcome.platform.merger.rules.{MergerRule, WorkPairMerger}
 
 /** If we have a pair of Sierra records:
   *
-  *   - One of which has a single Physical location ("physical work")
+  *   - One of which has a at least one item with at least a Physical location ("physical work")
```

Comment:
Would read better like this IMO:

```suggestion
  *   - One of which has a at least one item containing a Physical location ("physical work")
```